src/info.rs

```diff
@@ -28,10 +28,13 @@ pub const AJAZZ_VENDOR_ID_1: u16 = 0x5548;
 /// Product ID of Ajazz AKP153 Desk Controller
 pub const PID_AJAZZ_AKP153: u16 = 0x6674;
 
+/// Product ID of the "FHOOU" (front label) and "MiraBox HSV293S" (rear label). Seems just like an AKP153.
+pub const PID_MIRABOX_HSV293S: u16 = 0x6670;
+
 /// Product ID of Ajazz AKP815 Desk Controller
 pub const PID_AJAZZ_AKP815: u16 = 0x6672;
 
-/// Vendor ID of Ajazz AKP153E Desk Controller
+/// Vendor ID of Ajazz Stream Deck AKP153E
```

Comment:
This comment change is inconsistent with the other Ajazz devices; either change them all or not at all.